### PR TITLE
OCLOMRS-205: Fix UUID bug that pops up while creating a custom concept

### PR DESCRIPTION
**Pull request by emmabaye**
# JIRA TICKET NAME:
[Fix UUID bug that pops up while creating a custom concept](https://issues.openmrs.org/browse/OCLOMRS-205)

# Summary:
Fix the bug that displays an error message,"Concept could not be created, uuid must be unique", when a user wants to create a a custom concept.

### CODE REVIEW

**Comment by Michyjones** [comment]
LGTM

File: src/redux/actions/concepts/dictionaryConcepts.js [hunk 1]
```diff
@@ -185,12 +185,19 @@ export const createNewConcept = (data, dataUrl) => async (dispatch) => {
     dispatch(addConceptToDictionary(response.data.id, dataUrl));
     notify.show('creating concept, please wait...', 'warning', 3000);
   } catch (error) {
-    notify.show('concept could not be created, uuid must be unique', 'error', 3000);
     if (error.response) {
-      dispatch(isErrored(error.response.data, CREATE_NEW_CONCEPT));
+      const { response } = error;
+      notify.show(` There was error in creating a concept.
+ ${
```
Comment (dondrzzy) [inline src/redux/actions/concepts/dictionaryConcepts.js]: @emmabaye, is this properly formatted?

Comment (emmabaye) [reply src/redux/actions/concepts/dictionaryConcepts.js]: @dondrzzy Yes, it is.  ESlint throws no errors.

File: src/redux/actions/concepts/dictionaryConcepts.js [hunk 2]
```diff
@@ -185,12 +185,19 @@ export const createNewConcept = (data, dataUrl) => async (dispatch) => {
     dispatch(addConceptToDictionary(response.data.id, dataUrl));
     notify.show('creating concept, please wait...', 'warning', 3000);
   } catch (error) {
-    notify.show('concept could not be created, uuid must be unique', 'error', 3000);
     if (error.response) {
-      dispatch(isErrored(error.response.data, CREATE_NEW_CONCEPT));
+      const { response } = error;
+      notify.show(` There was error in creating a concept.
```
Comment (dkayiwa) [inline src/redux/actions/concepts/dictionaryConcepts.js]: The above grammar is not correct.

Comment (emmabaye) [reply src/redux/actions/concepts/dictionaryConcepts.js]: Thanks for the feedback. I will update it.

Comment (emmabaye) [reply src/redux/actions/concepts/dictionaryConcepts.js]: I have updated the description.

**Pull request merged**